MIEngine.mdd.nuspec

```diff
@@ -14,6 +14,7 @@
   <files>
     <file src="drop\Release\ReferenceAssemblies\*.xml" target="ref\dotnet" />
     <file src="drop\Release\ReferenceAssemblies\Microsoft.DebugEngineHost.dll" target="ref\dotnet" />
+    <file src="drop\Release\ReferenceAssemblies\Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.DesignTime.dll" target="ref\dotnet" />
     <file src="drop\Release\Microsoft.MICore.dll" target="ref\dotnet" />
```

Comment:
We should probably include Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.DesignTime.xml also.